docs/init.rst

```diff
@@ -42,6 +42,7 @@ Embrace functions and classmethods as a filter between reality and what's best f
 If you look for object serialization, there's a bunch of projects listed on our ``attrs`` extensions `Wiki page`_.
 Some of them even support nested schemas.
 
+.. _private_attributes:
```

Comment:
If I see it correctly, there's no narrative documentation for the new feature?


Would you mind adding it here and into examples.rst somewhere around the handling of private attributes?